source/v4/system.yaml

```diff
@@ -14,3 +14,22 @@
             $ref: "#/definitions/StatusOK"
         '400':
           $ref: '#/responses/BadRequest'
+
+  '/email/test':
+    post:
+      tags:
+        - system
+      summary: Send an test email
+      description: |
+        Send an test email
```

Comment:
Minor grammar change, can the summary be `Send test email` and the description be `Send a test email to make sure you have your email settings configured correctly.`